### ISSUE TITLE
Document how to contrib localizations

### ISSUE THREAD

**Issue by petri** [MacPass/MacPass#849]
I'd be willing to bootstrap Finnish localization. Is it sufficient to just copy over some of the existing ones and work from there?

It would be good if this information were in the FAQ as well.

**Comment by mstarke** [comment]
Nice to hear you want to contribute to MacPass. Take a look at the https://github.com/MacPass/MacPass/blob/master/CONTRIBUTE.md file. It contains a goof explanation on how to add a new localization.

**Comment by petri** [comment]
Thanks. It is quite a hassle to set up Xcode and XLIFF tool for someone who just wants to contribute translations. To attract translators and make it easier, how about using a translation service such as https://poeditor.com or https://www.transifex.com ? They offer free service for Open Source projects.

**Comment by mstarke** [comment]
You're right. Its rather complex. I did set up an account on https://lokalise.co but the workflow wasn't ideal so I stopped promoting it. I'll give your suggestions a try to see how they integrate.

**Comment by mstarke** [comment]
I've updated the Project in lokalise.co and try to re-import agains to see if I can integrate it. I tried to setup a project in POEditor but the UI in loklalise seem much more mature so I added you there as a contributor

**Comment by petri** [comment]
I've completed the Finnish translation in Lokalise to quite a high degree. So it could be used. However I am not sure whether the UI gives access to strings that should NOT be localized... ? For example should this be localized: %#@characters@ ?

**Comment by petri** [comment]
I cannot download the translations I added. I even installed the cli tool but seems I have no sufficient permissions. Can you please give my Lokalise account proper rights to download my contribution?

**Comment by mstarke** [comment]
I've given you download capabilities. Sorry for not giving any feedback. Generally any string can be localized but some of them are only placeholders in the UI files that will get changed by code so no need to change them. I should update the comments to clarify the ones that get changed by code.
